CONTRIBUTING.md

```diff
@@ -104,8 +104,8 @@ black manually to format files and commit them.
 
 ```bash
 # This should autoformat the files
-isort -rc .
 black .
+isort --profile black .
```

Comment:
In addition to this change, we have to modify also : 
- https://github.com/pytorch/ignite/blob/f51320a33571217818a3b429551b4bcdd97835f4/.github/workflows/code-style.yml#L33
- https://github.com/pytorch/ignite/blob/f51320a33571217818a3b429551b4bcdd97835f4/.github/workflows/unit-tests.yml#L85

Reply:
Oh right, the Actions workflow. Will do that, thanks!